schemas/oracle.py

```diff
@@ -0,0 +1,83 @@
+# zschema sub-schema for zgrab2's oracle module
+# Registers zgrab2-oracle globally, and oracle with the main zgrab2 schema.
+from zschema.leaves import *
+from zschema.compounds import *
+import zschema.registry
+
+import schemas.zcrypto as zcrypto
+import schemas.zgrab2 as zgrab2
+
+# Get a dict of all keys, mapping key -> true 
+def flagsSet(keys):
+    return SubRecord(dict((key, Boolean()) for key in keys))
```

Comment:
Arguably more Pythonic to write `SubRecord({key: Boolean() for key in keys})`, but this doesn't really matter.